src/test/java/pro/trevor/tankgame/rule/LootingTargetTest.java

```diff
@@ -20,20 +21,19 @@
 import pro.trevor.tankgame.rule.impl.shared.PlayerRules;
 
 
-class GoldLootTransfer implements ITriConsumer<State, GenericTank, AttributeContainer> {
-    public void accept(State state, GenericTank tank, AttributeContainer target) {
-        // Just set the gold to the other tank's value this method just exists to verify that we called the consumer
-        tank.put(Attribute.GOLD, target.getUnsafe(Attribute.GOLD));
-        target.put(Attribute.GOLD, 0);
-    };
-}
-
-
 public class LootingTargetTest extends LootActionTestHelper {
+    private static class GoldLootTransfer implements ITriConsumer<PlayerRuleContext, GenericTank, AttributeContainer> {
```

Comment:
Private static class with one function that is only used as a functional interface... Couldn't this just be a function?